drivers/src/lms.rs

```diff
@@ -301,6 +306,66 @@ impl Lms {
         Ok(HashValue::from(digest))
     }
 
+    #[cfg(feature = "hw-latest")]
+    fn hash_chain<const N: usize>(
+        &self,
+        sha256_driver: &mut impl Sha256Alg,
+        wnt_prefix: &mut [u8; 55],
+        coeff: u8,
+        params: &LmotsParameter,
+        tmp: &mut HashValue<N>,
+    ) -> CaliptraResult<HashValue<N>> {
+        let iteration_count = ((1u16 << params.w) - 1) as u8;
+
+        if coeff < iteration_count {
+            let mut digest = Array4x8::default();
+            let mut hasher = sha256_driver.digest_init()?;
+            wnt_prefix[Self::ITER_COUNTER_OFFSET] = coeff;
+            let mut i = Self::TMP_OFFSET;
+            for val in tmp.0.iter().take(N) {
+                wnt_prefix[i..i + 4].clone_from_slice(&val.to_be_bytes());
+                i += 4;
+            }
+            //set n_mode: 1 for n=32, and 0 for n=24
+            let mut n_mode: bool = false;
+            if params.n == Self::WNTZ_MODE_SHA256 {
+                n_mode = true;
+            }
+            hasher.update_wntz(&wnt_prefix[0..Self::TMP_OFFSET + N * 4], params.w, n_mode)?;
+            hasher.finalize_wntz(&mut digest, params.w, n_mode)?;
+            *tmp = HashValue::<N>::from(digest);
+        }
+        Ok(*tmp)
+    }
+
+    // This operation is accelerated in hardware by RTL1.1.
+    #[cfg(not(feature = "hw-latest"))]
+    fn hash_chain<const N: usize>(
+        &self,
+        sha256_driver: &mut impl Sha256Alg,
+        wnt_prefix: &mut [u8; 55],
```

Comment:
This values keeps coming up in the code. Let's define an informative constant for this.

Reply:
done